src/components/BookmarkCard/BookmarkCard.scss

```diff
@@ -66,7 +66,14 @@
 
         color: $color-grey-700;
 
-        @include multi-line-ellipsis(2);
+        &,
+        &--isTitleShort {
+            @include multi-line-ellipsis(3);
+        }
+
+        &--isDescriptionShort {
+            @include multi-line-ellipsis(4);
+        }
```

Comment:
These selectors look weird inside BEM structure. Maybe better to call them `--short-title` and `--short-description`?

Reply:
Agree with this.